s3/config.go

```diff
@@ -101,8 +107,18 @@ func newS3Client(config stow.Config) (client *s3.S3, endpoint string, err error)
 		authType = "accesskey"
 	}
 
+	httpClient := http.DefaultClient
+	skipSSLVerify, ok := config.Config(ConfigInsecureSkipSSLVerify)
```

Comment:
You could make this one line `if skip... (); ok && skip...`, but I'm not sure that's cleaner

Reply:
👍 - I'll make the change.